app/view_models/auctions_index_view_model.rb

```diff
@@ -1,29 +1,30 @@
-class AuctionsIndexViewModel < Struct.new(:current_user, :auctions_query)
-  def auctions
-    @auctions ||= auctions_query.map { |auction| AuctionViewModel.new(current_user, auction) }
+class AuctionsIndexViewModel
+  attr_reader :auctions, :current_user
+
+  def initialize(auctions:, current_user:)
+    @auctions = auctions.to_a
```

Comment:
BOOM fixed at 36c01ba

Reason `to_a` was needed was that `count` does not work properly with more than one SQL condition http://stackoverflow.com/a/18949440/1019369

solution was to use AR query rather than select on an array